configurations/zerotoken_nodes/zerotoken_node_i4ilarge_1_num_single_dc.yaml

```diff
@@ -0,0 +1,3 @@
+n_db_zero_token_nodes: 1
+zero_token_instance_type_db: 'i4i.large'
```

Comment:
it looks yes: I opened issue to get clear answer https://github.com/scylladb/scylladb/issues/21321. Also in next PR it could fixed with additional sct config params related to zero nodes allowing to config the ebs storage for zero nodes separately